dask_sql/physical/rex/core/call.py

```diff
@@ -169,11 +188,10 @@ def div(self, lhs, rhs):
         # We do not need to truncate in this case
         # So far, I did not spot any other occurrence
         # of this function.
-        if isinstance(result, datetime.timedelta):
+        if isinstance(result, (datetime.timedelta, np.timedelta64)):
             return result
```

Comment:
Could we add a test to cover this case? I ran through the test suite with a `breakpoint` and nothing hit this case